editor/import/resource_importer_texture_atlas.cpp

```diff
@@ -240,10 +255,25 @@ Error ResourceImporterTextureAtlas::import_group_file(const String &p_group_file
 		} else {
 			pack_data.is_mesh = true;
 
+			const int shrink_mask = options["shrink_mask"];
+			const int grow_mask = options["grow_mask"];
+			const float simplification = options["simplification"];
+
 			Ref<BitMap> bit_map;
 			bit_map.instantiate();
-			bit_map->create_from_image_alpha(image);
-			Vector<Vector<Vector2>> polygons = bit_map->clip_opaque_to_polygons(Rect2(0, 0, image->get_width(), image->get_height()));
+			bit_map->create_from_image_alpha(image, 0);
```

Comment:
The change to 0 threshold is as the default of 0.1 can create sharp edges of the transparent gradient